lib/OpenQA/CLI/api.pm

```diff
@@ -26,16 +26,30 @@ sub command ($self, @args) {
       'd|data=s' => \$data,
       'f|form' => \my $form,
       'j|json' => \my $json,
+      'H|hook-scripts=s' => \my $hooksrc,
```

Comment:
Here it is `--hook-scripts` and in the example before just `--hook-script`. I guess it should be the latter because being able to invoke one script must be enough.